benchexec/baseexecutor.py

```diff
@@ -75,8 +92,8 @@ def _start_execution(
         parent_cleanup_fn,
     ):
         """Actually start the tool and the measurements.
-        @param parent_setup_fn a function without parameters that is called in the parent process
-            immediately before the tool is started
+        @param parent_setup_fn a function that is called in the parent process
+            immediately before the tool is started. The keyword-arguments passed may differ in subclasses.
```

Comment:
I think beyond that we need documentation what parameters may actually be passed, either here or in the respective classes.